sktime/forecasting/ttm.py

```diff
@@ -321,22 +331,35 @@ def _fit(self, y, X, fh):
 
         # Get Training Configuration
         training_args = TrainingArguments(**self._training_args)
+        if not hasattr(self, "_model"):
```

Comment:
This else/if would not be required if we also have some kind of default tuner, which just tunes the default model, without applying PEFT.